### ISSUE TITLE
Groupes de l'éditeur ne correspondent pas à l'ORBAT en jeu

### ISSUE THREAD

**Issue by ImplicitDeny** [team-gsri/LM-Warfare#8]
Une mise en adéquation est nécessaire.

**Comment by ImplicitDeny** [comment]
Problème soulevé par @AkenoSyh mais semble déjà corrigé, les groupes du mod sont bien ceux de la Daily Warfare. A préciser...

**Comment by ArwynFr** [comment]
En tout cas il y a un écart entre l'orbat et la DW, donc un des deux est à mettre à jour.
3 ingénieurs semble pertinent pour une cavalerie blindée (3 personnels dans un char).

**Comment by ImplicitDeny** [comment]
Oui c'est probablement un oubli sur la DW, parce que le groupe portant le nom "Logistique" dans le mod (nom à revoir, certainement) comporte bel et bien trois ingénieurs.

**Comment by ArwynFr** [comment]
D'ailleurs j'ai vu une erreur dans les logs parce qu'il y a un décalage entre le nombre de slots et de joueurs. Par contre c'est peut-être plus dû à la suppression du Zeus ?

**Comment by ImplicitDeny** [comment]
Je pense pas, plutôt une erreur de config serveur. De mémoire il y a 16 places sur le serveur, or la DailyWarfare comporte 6x2 + 4x3 + 2x2 = 28 slots, ça ne peut pas être lié à la simple suppression d'un slot Zeus.

**Comment by ArwynFr** [comment]
Non, ça n'a rien à voir avec le nombre de joueurs max connectés (config serveur, valeur 16).

C'est un problème de décalage entre le nombre de slots de la **mission** (ça doit se configurer dans les options générales de la mission je pense) qui ne correspond pas au nombre de personnages avec la case « jouable » cochée. Ça donne un message d'erreur du style « 28 slots sur la mission mais seulement 27 personnages jouables ».

**Comment by ImplicitDeny** [comment]
J'avoue que j'ai pas trop d'idée O_o

**Comment by ArwynFr** [comment]
Le message exact :
`12:37:30 Mission LM_DailyWarfare_2_3.Altis: Number of roles (27) is different from 'description.ext::Header::maxPlayer' (28)`
